src/page-modules/contact/means-of-transport/means-of-transport-form-selector.tsx

```diff
@@ -0,0 +1,63 @@
+import { PageText, useTranslation } from '@atb/translations';
+import { SectionCard } from '../components/section-card';
+import { RadioInput } from '../components/input/radio';
+import { ContextProps } from './means-of-transport-form-machine';
+import { machineEvents } from '../machineEvents';
+
+type MeansOfTransportFormSelectorProps = {
+  state: {
+    hasTag(arg0: string): boolean | undefined;
+    context: ContextProps;
+  };
+  send: (event: typeof machineEvents) => void;
+};
+
+export const MeansOfTransportFormSelector = ({
```

Comment:
I think this is a good solution that we should use for the ticket-control page as well (in a separate PR)